.github/workflows/tag-publish.yaml

```diff
@@ -0,0 +1,114 @@
+name: Bump, Tag, and Publish
```

Comment:
I'd actually change this to say "Generate BEE Build" or something, because it doesn't do any real publishing (and the UI isn't actually really versioned by the bumper tags, either)